src/worker.js

```diff
@@ -23,6 +24,19 @@ function start() {
     }
   });
   scanQueue.process(maxJobsPerWorker, (job) => processFile(job.data.key));
+
+  reconciliationQueue.on('failed', (job, error) => auditLogger
+    .error(`${job.data.key}: Legacy report reconciliation failed with error ${error}`));
+  reconciliationQueue.on('completed', () => logger
+    .info('Legacy report reconciliation completed successfully'));
+  reconciliationQueue.process('legacyReports', async (job) => {
+    logger.info(`starting ${job}`);
+    try {
+      await reconcileLegacyReports();
+    } catch (e) {
```

Comment:
Does catching this error prevent the `on('failed'` callback from being run? Is there a reason that'd be better?

Reply:
it was just so I could see the error message in the console when debugging. It shouldn't catch it in prod. nice catch there.